### DISCUSSION TITLE
ONNX model

### DISCUSSION THREAD

**Discussion by shrestha-pranita** [speechbrain/speechbrain#1677] [Q&A] [answered]
Is there onnx model for VAD and speaker diarization?

**Comment by anautsch** [comment]
While speechbrain has models for these tasks, fbanks and stft are (as today) not exportable in pytorch.
There might be batch-independent computations in a future release (to separate from the model, e.g. fbanks & stft computations).

**Comment by TParcollet** [comment, marked as answer]
I concur with @anautsch ... Any model relying on torch stft can not be exported in onnx right now. This is a PyTorch issue, and they are aware of it. I guess it should be fixed soon, @mravanelli any more info on that topic?
